sixtyfps_runtime/corelib/items.rs

```diff
@@ -991,6 +991,12 @@ pub struct PropertyAnimation {
     pub easing: crate::animations::EasingCurve,
 }
 
+impl Default for PropertyAnimation {
+    fn default() -> Self {
+        Self { duration: 0, loop_count: 1, easing: crate::animations::EasingCurve::default() }
```

Comment:
I think Rust generated code and the interpreter call this default function, but C++ generated code will not. Maybe the default needs to go into builtins.60?

Reply:
Is that possible already?

All tests still passed for me, but looks like I somehow missed the C++ tests:-/

Reply:
It’s not your fault :). I’m not sure if there are tests for loop-count that use a .60 file.

Reply:
Found it. Now some more different tests fail though:-/